src/Data/LinkHeaders.php

```diff
@@ -115,18 +115,18 @@ public function makeUnique()
         return $this;
     }
 
-    public function __toString()
+    public function __toString(): string
     {
         return trim(collect($this->getLinkProvider()->getLinks())
-            ->map([static::class, 'linkToString'])
+            ->map([self::class, 'linkToString'])
```

Comment:
My preference here is to use static since if it is ever needed this class could be extended, overriding the linkToString function and then it would call the function on the class extending this one instead of still executing the function of this class itself